src/TypographyChart.svelte

```diff
@@ -377,7 +378,9 @@
   }
 
   function getLayers(style) {
-    const layers = expandLayers(style.layers.filter(layer => layer.type === 'symbol'));
+    const layers = $displayLayersStore.layers.filter(
```

Comment:
This is basically the only thing that changed here. I did _not_ add in the warnings for combination limits being hit here yet since it is more complex in Mapbox GL, but curious what you think. Should we:
- Handle as followup
- Handle here with a different indicator/warning
- Handle here with the same dot indicator implemented in GL